compiler/test/dotty/tools/TestSources.scala

```diff
@@ -43,7 +43,9 @@ object TestSources {
       .filter(_.nonEmpty)
       .toList
 
-    assert(list.nonEmpty)
+    if (list.isEmpty)
+      println(s"$path is empty")
```

Comment:
Why this change?

Reply:
To be able to remove all blacklisted item and run the tests to see which one actually fail.